.idea/.gitignore

```diff
@@ -0,0 +1,8 @@
+# Default ignored files
+/shelf/
+/workspace.xml
+# Editor-based HTTP Client requests
+/httpRequests/
+# Datasource local storage ignored files
+/dataSources/
+/dataSources.local.xml
```

Comment:
This file could again be "removed" from the repository.
It has probably been added before the `.gitignore` was updated :)

Reply:
Including .idea incorrectly. Fixing that.